recipes/recipes_emscripten/glib/recipe.yaml

```diff
@@ -24,6 +24,9 @@ requirements:
   - ${{ compiler('cxx') }}
   - meson
   - packaging
+  - zlib
```

Comment:
Are these packages (zlib, libffi, pcre2) needed in the build environment?

Reply:
hmmm, once sec ... confirming !

Reply:
No they won't be required.